dist/placeholder-buster.txt

```diff
@@ -250,9 +250,13 @@ gadgethacks.com##.whtaph-floatbox
 
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/1#issuecomment-450592252
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/33#issuecomment-506915515
-tomshardware.*##.p-u-md-1-3.p-u-1:has(div[class*="block-ads"])
-tomshardware.*###in_article
-tomshardware.*###below_the_article
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/57
+tomshardware.com##.p-u-md-1-3.p-u-1:has(div[class*="block-ads"])
+tomshardware.com###in_article
+tomshardware.com###below_the_article
+tomshardware.com##.slot-top_of_sidebar
+tomshardware.com##.slot-before_popular_box
+tomshardware.com##.slot-after_popular_box
```

Comment:
So the wildcard is no longer needed?

Reply:
`tomshardware.co.uk` appears to have been replaced by `www.tomshardware.com/uk`, so that is indeed correct.